lib/matplotlib/mlab.py

```diff
@@ -629,10 +629,9 @@ def _single_spectrum_helper(x, mode, Fs=None, window=None, pad_to=None,
         version of the segment.
 
     sides : {'default', 'onesided', 'twosided'}
-        Specifies which sides of the spectrum to return.  Default gives the
-        default behavior, which returns one-sided for real data and both
-        for complex data.  'onesided' forces the return of a one-sided
-        spectrum, while 'twosided' forces two-sided.
+        Which sides of the spectrum to return. 'default' is one-sided for real
```

Comment:
is it possible to have bulleted list?
* 'default': one-sided for real data and two-sided for complex data.
* 'onesided' forces the return of a one-sided spectrum
* 'twosided' forces two-sided

Reply:
That just seems very verbose when the interpretation of 'onesided' and 'twosided' are actually pretty obvious (only 'default' really needs an explanation).

Reply:
Agree it's verbose, but I think bulleted lists are easier to parse especially here where it becomes a left is the key, right is the meaning list.

Reply:
I would actually write the whole thing as
> Whether to return a 'onesided' or 'twosided' spectrum.  The 'default' is a 'onesided' spectrum for real data and 'twosided' for complex data.

"'foo' means foo" just reads horribly to me.

Reply:
But here the idea is 'foo' always forces a return of that type? So 'twosided' returns 'twosided' even on real data?

Reply:
Yes?

Reply:
So your preferred wording leaves out that forcing behavior.

Reply:
I honestly fail to see the semantic content of the word "forcing" here; what ambiguity would this word resolve?  What other behavior do you think the users could imagine if that word was removed?

Reply:
What I mean by forcing is that the keyword chosen == type returned (and I agree/don't think forcing is even the right word here). My concern is not about behaviors that users can imagine, it's about users potential uncertainty about what this argument does if it's not explicitly spelled out. But also, won't hold up the PR on this so I'll merge if you're done w/ changes.

Reply:
I... genuinely don't understand what is the potential uncertainty that you refer to. In any, if you're fine with the changes here I think this PR can go in as is.